app/views/layouts/application.html.slim

```diff
@@ -4,9 +4,9 @@ html
     title LearnITGirl
     meta charset="utf-8"
     meta content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0" name="viewport"
-    link href='https://fonts.googleapis.com/css?family=Roboto:400,100,700' rel='stylesheet' type='text/css'
+    link href='https://fonts.googleapis.com/css?family=Hind' rel='stylesheet' type='text/css'
     meta content="yes" name="apple-mobile-web-app-capable"
-    meta content="black" name="apple-mobile-web-app-status-bar-style"
+    link rel="shortcut icon" type="image/png" href="assets/favicon.png"
```

Comment:
as rails cache the image and add hash to it, it won't be able to find this image - you need to define path using rails helper `href="#{asset_url('favicon.png')}"`

Reply:
No problem. It'll be fixed.